lib/geordi/chromedriver_updater.rb

```diff
@@ -6,9 +6,12 @@
 
 module Geordi
   class ChromedriverUpdater
+    class ProcessingError < StandardError; end
+
     VERSIONS_PER_MILESTONES_URL = "https://googlechromelabs.github.io/chrome-for-testing/latest-versions-per-milestone-with-downloads.json"
 
     def run(options)
+      interaction_method = (options[:exit_on_failure] == true) ? :note : :fail
```

Comment:
I'd like to see this in the rescue block, if possible. Also, I'd change it to:
```suggestion
      interaction_method = options.exit_on_failure? ? :fail : :warn
```
/edit: the syntax `options.exit_on_failure?` might only work within Thor or with a Thor options hash. If it complicates Test, you may as well stick to your classic syntax – but please mind the twisted method names!